### PR TITLE
Reorganize notebooks into categories

### PR DESCRIPTION
**Pull request by mrmer1**
Categorize notebooks into 3 categories and move the relevant notebooks into the respective directories.
- LLM University
- Guides
- Enterprise

Update main and directory-level readmes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,160 +1,12 @@
...
-### 23. How to Fine-Tune a Model for Cohere Chat
-[<a href="https://github.com/cohere-ai/notebooks/blob/main/notebooks/Fine_tuning_Models_for_the_Chat_Endpoint.ipynb
-">Notebook</a> | <a href="https://colab.research.google.com/github/cohere-ai/notebooks/blob/main/notebooks/Fine_tuning_Models_for_the_Chat_Endpoint.ipynb
-">Colab</a>]
-
-<img src="https://raw.githubusercontent.com/cohere-ai/notebooks/main/notebooks/images/chat-finetune.png" style="width:100%; max-width:400px" alt="How to Fine-Tune a Model for Cohere Chat" />
\ No newline at end of file
+| Category | Description |
+| --- | --- |
+| [LLM University](notebooks/llmu/readme.md) | Guides for getting started with Cohere, starting with basic usage and progressing to advanced topics. The code companion to the full [LLM University course](https://llm.university/).|
```
Comment (jalammar) [inline README.md:10]: I think this link should be to notebooks/llmu/ and not to notebooks/llmu/readme.md. That way readers can see the list of notebooks in addition to the contents of the readme file.

File: README.md [hunk 2]
```diff
@@ -1,160 +1,12 @@
...
+| [Guides](notebooks/guides/readme.md) | Tutorials and step-by-step guides covering a range of topics, providing practical guidance and code examples.
```
Comment (jalammar) [inline README.md:11]: Same for this link

File: notebooks/llmu/readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# LLM University
+
+Here you will find the notebooks companion to the [LLM University course](https://llm.university/). It starts with the basics and progresses into more advanced topics. If you are new to Cohere, these guides are a great way to get started.
+
+The recommended approach is to go through the course and follow along in these notebooks. Having said that, you can still get a reasonable level of understanding if you only have time to go through the notebooks.
+
+# Contents
+
+## Text Representation
```
Comment (jalammar) [inline notebooks/llmu/readme.md:9]: Can we link to the Module (not necessarily every lesson, but just to the modules in LLMU)?

So something like:

## [Text Representation](https://docs.cohere.com/docs/intro-text-representation)

**Review by jalammar — COMMENTED** [review]
I sense that Guides should be first in the list before LLM University. It feels like LLM university notebooks would be best reached by direct links from LLMU lessons to the notebook.

It's a big improvement. Left a couple of comments. Otherwise looks good to me.

**Comment by mrmer1** [comment]
Thanks Jay. I've updated it based on your recommendations.

**Pull request merged**